File: Dashboard Template/vyzor-dotnet/mvc/final/Controllers/ApplicationsController.cs
using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using vyzor.Models;

namespace vyzor.Controllers;

public class ApplicationsController : Controller
{
    private readonly ILogger<ApplicationsController> _logger;

    public ApplicationsController(ILogger<ApplicationsController> logger)
    {
        _logger = logger;
    }

    [Route("/chat")]
    public IActionResult chat()
    {
        return View("chat");
    }

    [Route("/file-manager")]
    public IActionResult file_manager()
    {
        return View("file_manager");
    }

    [Route("/full-calendar")]
    public IActionResult full_calendar()
    {
        return View("full_calendar");
    }

    [Route("/gallery")]
    public IActionResult gallery()
    {
        return View("gallery");
    }

    [Route("/mail-settings")]
    public IActionResult mail_settings()
    {
        return View("mail_settings");
    }

    [Route("/mail")]
    public IActionResult mail()
    {
        return View("mail");
    }

    [Route("/sweet-alerts")]
    public IActionResult sweet_alerts()
    {
        return View("sweet_alerts");
    }

    [Route("/task-kanban-board")]
    public IActionResult task_kanban_board()
    {
        return View("task_kanban_board");
    }

    [Route("/task-list-view")]
    public IActionResult task_list_view()
    {
        return View("task_list_view");
    }

    [Route("/to-do-list")]
    public IActionResult to_do_list()
    {
        return View("to_do_list");
    }

}

File: Dashboard Template/vyzor-symfony/final/src/Controller/ApplicationsController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class ApplicationsController extends AbstractController
{

    #[Route('/chat', name: 'app_chat')]
    public function chat(): Response
    {
        return $this->render('applications/chat.html.twig');
    }

    #[Route('/file-manager', name: 'app_file_manager')]
    public function fileManager(): Response
    {
        return $this->render('applications/file-manager.html.twig');
    }

    #[Route('/full-calendar', name: 'app_full_calendar')]
    public function fullCalendar(): Response
    {
        return $this->render('applications/full-calendar.html.twig');
    }

    #[Route('/gallery', name: 'app_gallery')]
    public function gallery(): Response
    {
        return $this->render('applications/gallery.html.twig');
    }

    #[Route('/mail-settings', name: 'app_mail_settings')]
    public function mailSettings(): Response
    {
        return $this->render('applications/mail-settings.html.twig');
    }

    #[Route('/mail', name: 'app_mail')]
    public function mail(): Response
    {
        return $this->render('applications/mail.html.twig');
    }

    #[Route('/sweet-alerts', name: 'app_sweet_alerts')]
    public function sweetAlerts(): Response
    {
        return $this->render('applications/sweet-alerts.html.twig');
    }

    #[Route('/task-kanban-board', name: 'app_task_kanban_board')]
    public function taskKanbanBoard(): Response
    {
        return $this->render('applications/task-kanban-board.html.twig');
    }

    #[Route('/task-list-view', name: 'app_task_list_view')]
    public function taskListView(): Response
    {
        return $this->render('applications/task-list-view.html.twig');
    }

    #[Route('/to-do-list', name: 'app_to_do_list')]
    public function toDoList(): Response
    {
        return $this->render('applications/to-do-list.html.twig');
    }

}
